api/v1beta1/glanceapi_webhook.go

```diff
@@ -0,0 +1,93 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	"k8s.io/apimachinery/pkg/runtime"
+	ctrl "sigs.k8s.io/controller-runtime"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/webhook"
+)
+
+// GlanceAPIDefaults -
+type GlanceAPIDefaults struct {
+	ContainerImageURL string
+}
+
+var glanceAPIDefaults GlanceAPIDefaults
+
+// log is for logging in this package.
+var glanceapilog = logf.Log.WithName("glanceapi-resource")
+
+// SetupGlanceAPIDefaults - initialize Glance spec defaults for use with either internal or external webhooks
```

Comment:
nit: `s/initialize Glance/initialize GlanceAPI/`

Reply:
Good catch, updating it

Reply:
Done